dttools/src/catalog_query.h

```diff
@@ -20,6 +20,10 @@ Query the global catalog server for server descriptions.
 #define CATALOG_HOST (getenv("CATALOG_HOST") ? getenv("CATALOG_HOST") : CATALOG_HOST_DEFAULT )
 #define CATALOG_PORT (getenv("CATALOG_PORT") ? atoi(getenv("CATALOG_PORT")) : CATALOG_PORT_DEFAULT )
 
+/* Name for the automatically generated ids for records generated by
```

Comment:
Suggest just dropping this additional symbol and use "uuid" throughout, since it's going to be a strong convention for backwards compatibility..